language.md

```diff
@@ -175,6 +174,20 @@ name, which will be exported as `shesmu_user_fastqc`, must be unique in the
 program. After the name is the help text. Inside the braces, labels can be
 specified; the values must be strings.
 
+Additionally, for more serious debugging, the data passing through an olive can be dumped:
+
+    Run fastqc
+      Where workflow == "BamQC 2.7+"
+      Dump metatype, ius To some_file
+      With {
+        memory = 4Gi,
+        input = path
+      }
+
+The specified expressions will be dumped to a file. The file is defined by a
+dumper. If no dumper exists, the output is sent nowhere. This makes it possible
+to leave these in production systems with a problem.
```

Comment:
"To leave these in production systems with a problem" = "To leave these in production system to help debug a problem"?

Reply:
I can totally words today.